testonly/loglb/main.go

```diff
@@ -65,6 +66,7 @@ func newRandomLoadBalancer(serverCfg string) (*randomLoadBalancer, error) {
 	return &lb, nil
 }
 
+// TODO(somebody): This function name collides with a builtin function and should be renamed
```

Comment:
I think this is fine. We'll use it as `lb.close()`, not `close(lb)`, so it's not ambiguous.